.github/workflows/OCV-WinPack-4.x-W10.yaml

```diff
@@ -0,0 +1,148 @@
+name: OCV WinPack:4.x W10
+
+on:
+  pull_request:
+    branches:
+      - main
+    paths:
+      - '.github/workflows/OCV-WinPack-4.x-W10.yaml'
+  workflow_dispatch:
+  schedule:
+    - cron: '0 0 * * *'
+
+env:
+  BRANCH_NAME: 4.x
+
+jobs:
+  Build:
+    runs-on: opencv-cn-win
+    defaults:
+      run:
+        shell: cmd
+    steps:
+    - name: Clean
+      timeout-minutes: 60
+      run: cd ${{ github.workspace }} && rm -rf *
+    - name: Fetch opencv
+      timeout-minutes: 60
+      run: cd ${{ github.workspace }} && git clone --branch ${{ env.BRANCH_NAME }} --reference %GIT_CACHE%\opencv.git git@github.com:opencv/opencv.git
+    - name: Cmake OpenCV
+      timeout-minutes: 60
+      env:
+        EXTRA_CMAKE_OPTIONS: '-DWITH_OBSENSOR=OFF -DOPENCV_PYTHON_INSTALL_PATH=python -DCMAKE_INSTALL_PREFIX=${{ github.workspace }}/install -DBUILD_SHARED_LIBS=ON -DBUILD_PERF_TESTS=OFF -DBUILD_TESTS=OFF -DBUILD_EXAMPLES=OFF -DINSTALL_PDB=ON -DINSTALL_PDB_COMPONENT_EXCLUDE_FROM_ALL=OFF -DCPU_BASELINE=SSE3 -DOPENCV_INSTALL_DATA_DIR_RELATIVE=${{ github.workspace }}\\opencv -DWITH_LAPACK=OFF -DVIDEOIO_PLUGIN_LIST=all -DINSTALL_CREATE_DISTRIB=ON -DBUILD_DOCS=OFF -DOPENCV_GENERATE_SETUPVARS=ON -DWITH_OPENCL=ON -DBUILD_opencv_python3=OFF -DBUILD_opencv_python2=OFF -DWITH_TBB=OFF -DWITH_CUDA=OFF -DBUILD_opencv_java=OFF'
```

Comment:
`-DWITH_OBSENSOR=OFF` is set temporary, while https://github.com/opencv/opencv/pull/22327 isn't merged.

Reply:
This pr has been merged.